app/serializers/keyword_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class KeywordSerializer
```

Comment:
How about adding the `V1` here as well @Goose97

Reply:
Fixed in 4b9c59c46f32fe39ec662e53caa1abf4d99a2d71